mailpoet/assets/js/src/automation/integrations/core/steps/if-else/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { select } from '@wordpress/data';
+import { __, _x, sprintf } from '@wordpress/i18n';
+import { blockMeta } from '@wordpress/icons';
+import { Edit } from './edit';
+import { storeName, StepType } from '../../../../editor/store';
+import { BranchBadge } from './branch-badge';
+import { Footer } from './footer';
+
+const keywords = [
+  __('wait', 'mailpoet'),
+  __('pause', 'mailpoet'),
+  __('delay', 'mailpoet'),
+  __('time', 'mailpoet'),
+];
```

Comment:
probably copy-pasted from the delay block.

Reply:
Thx. Fix: https://github.com/mailpoet/mailpoet/pull/5216/commits/8c6d27742f6f2693c6202ceb2765cfc4ff50b8cf